evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -909,23 +917,42 @@ def test_generate_pipeline_example(
         trans_primitives=["absolute"],
         return_types="all",
     )
+
+    X_train_t.index = y_train.index
     features_path = os.path.join(str(tmpdir), "features.json")
     ft.save_features(features, features_path)
-
+    ensembling = ensembling if not is_time_series(automl_type) else False
     aml = AutoMLSearch(
         X_train=X_train_t,
         y_train=y_train,
         problem_type=automl_type,
         optimize_thresholds=False,
-        max_iterations=5,
         problem_configuration=problem_configuration,
         features=features,
+        ensembling=ensembling,
     )
-    env = AutoMLTestEnv(automl_type)
-    with env.test_context(score_return_value={aml.objective.name: 1.0}):
-        aml.search()
 
-    pipeline = aml.get_pipeline(2)
+    if ensembling:
+        aml.search()
+    else:
+        env = AutoMLTestEnv(automl_type)
+        with env.test_context(score_return_value={aml.objective.name: 1.0}):
+            aml.search()
+
+    if ensembling:
+        for i in range(len(aml.full_rankings)):
+            pipeline = aml.get_pipeline(i)
+            if isinstance(
+                pipeline.estimator,
+                (StackedEnsembleClassifier, StackedEnsembleRegressor),
```

Comment:
For simplicity's sake, you can just check `isinstance(pipeline.estimator, StackedEnsembleBase)` here instead of checking for both!

Reply:
good idea!